### ISSUE TITLE
Can't progress past mod selection screen

### ISSUE THREAD

**Issue by maptap** [LoneGazebo/Community-Patch-DLL#3800]
_1. Mod version (i.e Date - 4/23):_
 12/15

_2. Mod list (if using Vox Populi only, leave blank):_

_3. Error description:_ 
By the select mod screen I select all 7 vox populi mods. I then click next and my cursor turns into the spinning world thing to show it was loading. After a few seconds I hear a click like I pressed something in game and the cursor turns back into an arrow. 

However, the screen does not change - it remains at the select mods screen with "loading please wait". It stays like this for ever - I have waited 10 minutes. 

Then I pull up the task manager to end Civ 5. After that when I start any game (civ 5 or otherwise) I get a message saying the computer cannot find a valid direct 3d device. I must restart my computer to get it working.  This does not happen with any other game - including witcher 3 and EU4.

I have tried reinstalling Civ 5 and vox populi to no effect.

Thanks!

[Logs.zip](https://github.com/LoneGazebo/Community-Patch-DLL/files/1601403/Logs.zip)

_4. Steps to reproduce (optional):_

---------------------------
Supporting information:
Please note that you can attach .zip files by dragging-and-dropping them. If possible, zip up all supporting data and post that way.

1. Log files (always attach your Logs folder, located at My Documents/My Games/Sid Meier's Civilization 5. Make sure you have enabled logging before experiencing an error! Go here to find out how: http://forums.civfanatics.com/showthread.php?t=487482):

2. Save game (always attach a save that was made a turn before the error; located at My Documents/My Games/Sid Meier's Civilization 5/ModdedSaves):

3. CvMiniDump.dmp file (attach if experiencing a game crash. Located at Program Files/Steam/steamapps/common/Sid Meier's Civilization V):

4. Screenshots (optional):

**Comment by LoneGazebo** [comment]
Make sure you are on the latest civ (.279).

G

**Comment by maptap** [comment]
How do I check that?

**Comment by maptap** [comment]
I checked. It is .279

**Comment by Infixo** [comment]
Why do you select 7 mods? Vox (standard, not 43civ) has 6 components: (1)-(5) plus UI (6a) OR (6b). 
Also, as usual try do close game, clear cache folder, and start a new.

**Comment by maptap** [comment]
I just realized you're right. I had also selected (7) Community Events (v 4) (made by Enginseer specifically for Vox Populi, I believe). 

Anyway, after clearing the cache and this time only selecting the vox populi I was able to start the game. The bug I reported only happens if I play with (7) Community Events (v 4), and it is still happening after clearing the cache. 

Should I repost this including (7) Community Events (v 4) on my mod list?
